### PR TITLE
EVG-6004: add command to update/force restart Jasper in user data script

### PR DESCRIPTION
**Pull request by Kimchelly**
JIRA: https://jira.mongodb.org/browse/EVG-6004

User data now gets latest Jasper CLI version and updates the config of the Jasper service on the host to reflect this.

### CODE REVIEW

File: model/host/hostutil.go [hunk 1]
```diff
@@ -230,16 +230,22 @@ func (h *Host) jasperExtractedFileName(config evergreen.JasperConfig) string {
...
+			h.FetchJasperCommandWithPath(config, "/bin"),
+			h.ForceReinstallJasperCommand(config),
+		}
 		// PowerShell nested quotation marks are handled by using two quotation
 		// marks.
-		quotedCmds := strings.Replace(cmds, "'", "''", -1)
+		quotedCmds := make([]string, 0, len(cmds))
+		for _, cmd := range cmds {
+			quotedCmds = append(quotedCmds, strings.Replace(cmd, "'", "''", -1))
+		}
```
Comment (tychoish) [inline model/host/hostutil.go]: does it make sense to put this in some kind of util function?

File: model/host/hostutil.go [hunk 2]
```diff
@@ -230,16 +230,22 @@ func (h *Host) jasperExtractedFileName(config evergreen.JasperConfig) string {
...
 		commands := []string{
 			"<powershell>",
 			fmt.Sprintf("%s -c '%s'", h.Distro.ShellPath, quotedCmds),
 			"</powershell>",
 		}
 		return strings.Join(commands, "\r\n")
```
Comment (tychoish) [inline model/host/hostutil.go]: should \r\n be in a constant? or is it a constant?

File: model/host/hostutil.go (hunk 1 above)
Comment (Kimchelly) [reply model/host/hostutil.go]: It seems like this is very specific to Powershell only, so unless there's some future use case for writing Powershell scripts programmatically, I think this is just a one-off thing.

File: model/host/hostutil.go (hunk 2 above)
Comment (Kimchelly) [reply model/host/hostutil.go]: There's no constant in the standard library for this. All the stdlib code that I've seen seems to use "\r\n" or define an unexported const.

**Pull request merged**